examples/ma_block_example.py

```diff
@@ -0,0 +1,74 @@
+import asyncio
+from flow_py_sdk import flow_client
+
+# -------------------------------------------------------------------------
+# Global variable
+# -------------------------------------------------------------------------
+access_node_host: str = "access.devnet.nodes.onflow.org"
+access_node_port: int = 9000
+
+# -------------------------------------------------------------------------
+# Rretrieve a block by ID Function
+# -------------------------------------------------------------------------
+async def get_block_by_id_example(block_id: bytes = None):
+    if(block_id == None):
+        block_id = '48338b132f33a7d8776adc777730c601e1d4f4ccb04261c066aad78269b14e9e'
+
+    # First Step : Create a client to connect to the flow blockchain
+    # flow_client function creates a client using the host and port
+    async with flow_client(
+            host = access_node_host, port = access_node_port
+        ) as client:
+            block = await client.get_block_by_i_d(
+                id = bytes.fromhex(block_id)
+            )
+            print("Block :\n")
+            print(block.__dict__)
+            print("\nget block by id : successfully done...")
```

Comment:
We want to encourage using the emulator for testing and experimenting, so the examples should be oriented towards having an emulator running locally.

Since the emulator is running locally the block ids are not necessarily constant from run to run (because of different emulator versions and configurations),I would suggest joining the by_height and by_id example.

It would also be great to use the existing example framework. Which would mean this example would become.
```suggestion
class GetBlockIdExample(Example):
    def __init__(self) -> None:
        super().__init__(tag="C.1.", name="GetBlockIdExample", sort_order=101)

    async def run(self, ctx: Config):
        # First Step : Create a client to connect to the flow blockchain
        # flow_client function creates a client using the host and port
        async with flow_client(
                host=ctx.access_node_host, port=ctx.access_node_port
        ) as client:
            block = await client.get_block_by_height(
                height=0
            )
            print("Block :\n")
            print(block.__dict__)
            print("\nget block by height : successfully done...")

            block = await client.get_block_by_i_d(
                id=block.id
            )
            print("Block :\n")
            print(block.__dict__)
            print("\nget block by id : successfully done...")
```
Which would mean the example can then be run as `poetry run examples C.1.`


If you disagree with anything, let me know.

Reply:
Understand.